apollo-livedata-support/src/main/java/com/apollographql/apollo/livedata/ApolloLiveDataResponse.kt

```diff
@@ -0,0 +1,80 @@
+package com.apollographql.apollo.livedata
+
+import com.apollographql.apollo.api.Error
+import com.apollographql.apollo.api.Response
+import com.apollographql.apollo.exception.ApolloException
+
+/**
+ * The ApolloLiveDataResponse class maps [Response] to [ApolloLiveDataResponse].
+ * There are Success, Failure, UnsuccessfulResponse, Complete sealed type classes.
+ */
+
+@Suppress("unused", "HasPlatformType")
+sealed class ApolloLiveDataResponse<out T> {
+
+    /**
+     * API Success response class.
+     *
+     * [data] is an Optional. (There are responses without data)
+     */
+    class Success<T>(var response: Response<T>) : ApolloLiveDataResponse<T>() {
+        var operation = response.operation()
+        var data: T? = response.data()
+        var dependentKeys: Set<String> = response.dependentKeys()
+        var fromCache: Boolean = response.fromCache()
```

Comment:
Sorry I might have provided misleading comment before.
I meant to remove all these fields and just leave `val response`.

Reply:
:white_check_mark: It fixed. Thank you!